src/context/ContextBuilder.tsx

```diff
@@ -39,14 +39,15 @@ function ContextBuilder(props: any) {
     return (
       <div className="context-cards">
         {topics?.length > 0 &&
-          topics.map((topic) => {
+          topics.map((topic, index) => {
```

Comment:
You can maybe change this index, to be the `topic.id` or `topic._id` if it exists. React doesn't want use to use map index's as the keys.

Reply:
done